experiment_scheduler/master/master.py

```diff
@@ -70,10 +70,13 @@ def __init__(self):
         # [Todo] Logging required
         # [TODO] need discussion about path and env vars
         self.task_managers_address: list = self.get_task_managers()
+        self.retry_task_list = []
```

Comment:
We use retry_task_list queue for saving retry task when there is no available task_manager.
But we delete this queue and change the logic of execute task.